### PR TITLE
Feat custom event renderer // Fix types

### PR DESCRIPTION
**Pull request by sebastienfi**
This fixes the typings on recent improvement https://github.com/acro5piano/react-native-big-calendar/pull/359

### CODE REVIEW

**Comment by acro5piano** [comment]
Thanks for this PR too. Seems that GitHub actions not works for forked repo.

File: src/CalendarBody.tsx [hunk 1]
```diff
@@ -189,7 +189,7 @@ export function _CalendarBody({
             ))}
             {dayJsConvertedEvents
               .filter(({ start }) =>
-                start.isBetween(date.startOf('day'), date.endOf('day'), null, '[)'),
+                (start as dayjs.Dayjs).isBetween(date.startOf('day'), date.endOf('day'), null, '[)'),
```
Comment (acro5piano) [inline src/CalendarBody.tsx:192]: Do you have any way to omit the castings `as dayjs.Dayjs`?
It should be more readable.

**Review by acro5piano — CHANGES_REQUESTED** [review]
Thanks very much!

**Comment by acro5piano** [comment]
@sebastienfi

Thank you for the PR. I've added a review for the typing.

https://github.com/acro5piano/react-native-big-calendar/pull/359

Sorry I thought GitHub actions checks typings, but it didn't. So if you fix type errors on that PR with this PR, I'd happy to merge them.

**Comment by sebastienfi** [comment]
@acro5piano Done. Let me know if anything else.

**Comment by acro5piano** [comment]
@sebastienfi Thanks!!

**Review by acro5piano — APPROVED** [review]

**Pull request merged**